timeless/__init__.py

```diff
@@ -7,18 +7,22 @@
 # pylint: disable=W0612
 import os
 from flask import Flask
-from flask_caching import Cache
+from timeless.cache import CACHE
 from timeless.db import DB
 from timeless.sync.celery import make_celery
 
-cache = Cache()
-
-
 def create_app(config):
     """Creates a new Timeless webapp given a config class"""
     app = Flask(__name__, instance_relative_config=True)
     app.config.from_object(config)
-    cache.init_app(app, config={"CACHE_TYPE": "redis"})
+    redis_config = {
+        "CACHE_TYPE": "redis",
+        "CACHE_REDIS_URL": app.config.get("REDIS_HOST")
+    }
+    CACHE.init_app(
+        app,
+        config=redis_config
```

Comment:
@ammaratef45 I expected a bit different, see for example https://github.com/gofynd/flask-full/blob/master/app/__init__.py#L47

We can implement something like:
```python
class StagingConfig(Config):
    REDIS_HOST = "redis://localhost:6379"
    RESULT_BACKEND = REDIS_HOST
    BROKER_URL = REDIS_HOST
    CACHE_SETTINGS = {
        "CACHE_TYPE": "redis",
        "CACHE_REDIS_URL": REDIS_HOST
    }
```
and then pass it to `Cache.init_app`: `CACHE.init_app(app, config=config.CACHE_SETTINGS)`.
So this way all settings will be in one place

Reply:
@vryazanov looks better 👍